fmn/util.py

```diff
@@ -36,3 +37,16 @@ def get_fas_email(config, username):
     except Exception:
         log.exception("Failed to get FAS email for %r" % username)
         return '%s@fedoraproject.org' % username
+
+
+def get_fasjson_email(config, username):
+    """ Return FASJSON email associated with a username. """
+    try:
+        fasjson = config["fasjson"]
+        client = fasjson_client.Client(url=fasjson.get('url'))
+        person = client.get_user(username=username).result
+
+        return person.get('emails')[0]
```

Comment:
not sure what the best way here is, fasjson will provide multiple emails and the data structure calling this functions expects one.

Reply:
Yeah getting the fist one seems a good default, even though the LDAP entries are not sorted (so the "first" in the list could be different between calls).